pkg/model/api/config/authentication.go

```diff
@@ -36,6 +36,7 @@ type OIDCProvider struct {
 	Scopes       []string          `json:"scopes" yaml:"scopes"`
 	Issuer       string            `json:"issuer" yaml:"issuer"`
 	DiscoveryURL string            `json:"discovery_url" yaml:"discovery_url"`
+	DisablePKCE  bool              `json:"disable_pkce" yaml:"disable_pkce"`
```

Comment:
can you update the doc about the config ?

Reply:
I actually did it in a next commit